parsing/location.ml

```diff
@@ -546,6 +547,22 @@ let lines_around_from_lexbuf
     lines_around ~start_pos ~end_pos ~seek ~read_char
   end
 
+(* Try to get lines from a phrase buffer fix error #7925 *)
```

Comment:
I don't think you should refer to the error that this code fixes. 
Just describe what the new code does -- the rest can be read from the git history and the Changes file.